publish.css

```diff
@@ -85,68 +174,76 @@ Last Update:
   --interactive-accent-rgb: #22a578;
   --vim-cursor: #22a578;
   /*  --text-selection:             add if needed */
-}
+  }
+
+h1,
+.markdown-rendered h1,
+.cm-line:not(.HyperMD-codeblock) .cm-header-1,
+.HyperMD-list-line .cm-header-1 {
+  font-variant-caps: var(--wws-header-font-caps-variant);
+  }
+
```

Comment:
Added this for H1 font variant. You can add more for h2-h6. Technically, the `.cm-line:not(.HyperMD-codeblock) .cm-header-1,` can be removed.